session4/src/com/itschool/homework/Challenge5.java

```diff
@@ -0,0 +1,36 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+public class Challenge5 {
+
+    public static void main(String[] args) {
+//
+//        int x = 10;
+//        int y = 11;
+//        //assignment and compound assignment operators
+//
+//        System.out.println(x*=5);
+//        System.out.println(x%=3);
+//        System.out.println(y%=3);
+//        System.out.println(x+=11);
+//        System.out.println(x-=1);
+//        System.out.println(x/=2);
```

Comment:
what's the deal with all this commented code?

Reply:
I removed the commented code, thank you!